lib/src/keyboard.dart

```diff
@@ -636,36 +636,36 @@ extension KeyboardInput on WidgetTester {
     await tester.pumpAndSettle();
   }
 
-  Future<void> pressCmdZ(WidgetTester tester) async {
-    await tester.sendKeyDownEvent(LogicalKeyboardKey.meta, platform: _keyEventPlatform);
-    await tester.sendKeyDownEvent(LogicalKeyboardKey.keyZ, platform: _keyEventPlatform);
-    await tester.sendKeyUpEvent(LogicalKeyboardKey.keyZ, platform: _keyEventPlatform);
-    await tester.sendKeyUpEvent(LogicalKeyboardKey.meta, platform: _keyEventPlatform);
+  Future<void> pressCmdZ() async {
```

Comment:
There are more methods that take a `tester` above this. Let's get rid of all of them.